spec/spec_helper.rb

```diff
@@ -37,8 +37,11 @@
     /codeclimate.com/, # For uploading coverage reports
     /chromedriver\.storage\.googleapis\.com/, # For fetching a chromedriver binary
   ],
+  net_http_connect_on_start: true
```

Comment:
I was getting the below error

`
     Failure/Error: TCPSocket.open(conn_addr, conn_port, @local_host, @local_port)

     Errno::EMFILE:
       Failed to open TCP connection to 127.0.0.1:9515 (Too many open files - socket(2) for "127.0.0.1" port 9515)
`
So I did some digging and [found this fix](https://stackoverflow.com/a/65946077). Hopefully this error does not happen again.